coverage_crawler/github.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+
+import os
+import shutil
+import subprocess
+
+
+def upload_to_github(report_path, git_user_name, git_password):
+    # Clone the repository if doesn't exist
+    if not os.path.isdir('coverage-crawler-reports'):
+        subprocess.run(['git', 'clone', 'https://github.com/coverage-crawler-updater/coverage-crawler-reports'])
+    os.chdir('coverage-crawler-reports')
+
+    # Remove the content of repository except of README
```

Comment:
We can remove the README too, the repository will only be accessed from https://rhcu.github.io/coverage-crawler-reports/, so the README is unneeded.